src/PhoXiInterface.cpp

```diff
@@ -19,6 +20,17 @@ namespace phoxi_camera {
         std::vector<PhoXiDeviceInformation> deviceInfo;
         auto dl = phoXiFactory.GetDeviceList();
         toPhoXiCameraDeviceInforamtion(dl, deviceInfo);
+
+        std::map<std::string, std::string> scannersIPs = PhoXiInterface::getScannersIPs();
+
+        for (auto &device : deviceInfo) {
+            if (scannersIPs.empty() || (scannersIPs.find(device.hwIdentification) == scannersIPs.end())) {
+                device.IPaddress = "unknown";
```

Comment:
This string ("unknown") is in "binpicking" repo forwarded to the dropdown list UI, so in current implementation its not translated. Is it ok, in this case? @simongrac @najlepsiwebdesigner